### ISSUE TITLE
[Issue] Login dialog still show

### ISSUE THREAD

**Issue by chipfc** [WiseClock/flarum-ext-login2see#1]
Hi,
My system:

 - flarum/core: ^0.1.0-beta.6
 - flarum-ext-login2see 0.1.1

Issue: 

 - I login to my forum, and click link in a post, and login dialog still show.

Please help me check it, thanks.

**Comment by xuxusheng** [comment]
@chipfc 
I've met the problem as you too.

it's caused by the code below :

![image](https://user-images.githubusercontent.com/15248446/29864533-9e73e898-8da5-11e7-951d-d6f85f78a2e5.png)

and you can check the option "Do not replace" to resolve it, although i think this is not the best solution.

![image](https://user-images.githubusercontent.com/15248446/29864431-4704b628-8da5-11e7-9dd7-e32b7c2bc545.png)

**Comment by Ralkage** [comment]
Lets get a PR to fix this issue, shall we? :)

**Comment by Justman10000** [comment]
@WiseClock

**Comment by Justman10000** [comment]
I don't think this is still being serviced! Especially since a [sequel by someone else](/jslirola/flarum-ext-login2seeplus) already exists!

**Comment by Justman10000** [comment]
@chipfc @xuxusheng @Ralkage

I don't think this is still being serviced! Especially since a [sequel by someone else](/jslirola/flarum-ext-login2seeplus) already exists!
